trl/templates/lm_model_card.md

```diff
@@ -11,9 +11,10 @@ It has been trained using [TRL](https://github.com/huggingface/trl).
 
 ```python
 from transformers import pipeline
+from transformers.testing_utils import torch_device
 
 question = "If you had a time machine, but could only go to the past or the future once and never return, which would you choose and why?"
-generator = pipeline("text-generation", model="{{ hub_model_id }}", device="cuda")
+generator = pipeline("text-generation", model="{{ hub_model_id }}", device=torch_device)
```

Comment:
It's just an example here, I'd keep it simple. the user can easily modify the device if needed

```suggestion
generator = pipeline("text-generation", model="{{ hub_model_id }}", device="cuda")
```